packages/redux/src/raffles/types/actions.types.ts

```diff
@@ -0,0 +1,194 @@
+import type * as actionTypes from '../actionTypes';
+import type { Action } from 'redux';
+import type {
+  BlackoutError,
+  Raffle,
+  RaffleEstimation,
+  RaffleParticipation,
+} from '@farfetch/blackout-client';
+import type { NormalizedSchema } from 'normalizr';
+
+/**
+ * Payload of a raffle
+ */
+type RaffleSuccessPayload = NormalizedSchema<
+  {
+    raffles: Record<Raffle['id'], Raffle>;
```

Comment:
This works, but you have a `RaffleEntity` that should be the type used for the `raffles` entities.